### PR TITLE
New translations

### PR DESCRIPTION
**Pull request by sseerrggii**

### CODE REVIEW

**Comment by sseerrggii** [comment]
Pre-release translations
Waiting for review 
I will test it after merge in staging

File: config/locales/ca.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ ca:
       common:
         created_at: Creat
         updated_at: Actualitzat
+      inquiry:
+        is_group: Demanda Grupal
```
Comment (enricostano) [inline config/locales/ca.yml]: Is the capital `G` needed? It's a bit ugly IMHO

**Comment by enricostano** [comment]
Please let's try to make no exceptions about:

1. testing in staging
2. merge

We need a clear and simple process. No `IF`, thanks.

**Comment by sseerrggii** [comment]
I thought we had already talked about it, translations is a pre-release process 

I need current develop to test translations

**Pull request merged**